src/gui/src/gui.tcl

```diff
@@ -137,6 +137,93 @@ proc save_image { args } {
   rename $options ""
 }
 
+sta::define_cmd_args "save_gif" {-start|-add|-end \
+                                 [-area {x0 y0 x1 y1}] \
+                                 [-width width] \
+                                 [-resolution microns_per_pixel] \
+                                 [-display_option option] \
+                                 [-delay delay] \
+                                 [path]
+} ;# checker off
```

Comment:
Why do we need the checker off?

Reply:
copy-paste from the `save_image` command